src/bin/pgcopydb/cli_restore.c

```diff
@@ -153,7 +153,6 @@ cli_restore_schema_getopts(int argc, char **argv)
 		{ "source", required_argument, NULL, 'S' },
 		{ "target", required_argument, NULL, 'T' },
 		{ "dir", required_argument, NULL, 'D' },
-		{ "schema", required_argument, NULL, 's' },
```

Comment:
@dimitri please let me know if that is ok to remove this line.

This particular option does not seem to exist anywhere in the codebase today. I suspect, this is a remnant of a removed argument.

Reply:
We have `--schema-name` option in `cli_list_db_getopts`. The trace log of `--schema` in there makes me think that we renamed it at some point.

Shall I rename this command to `--schema-name` and add the case block in the same function?

Reply:
I can't remember why we have `--schema` in there, seems ok to just remove it now as part of that cleanup PR. Thanks!